### DISCUSSION TITLE
Tomcat on Windows not in Inventory and no Vulnerabilities found

### DISCUSSION THREAD

**Discussion by martinzeifang** [wazuh/wazuh#15606] [General]
Hi everyone, 

for testing I installed a very old Tomcat Version (4.1.3) on a Windows System. I can access the tomcat website and wazuh found the installation: 
![image](https://user-images.githubusercontent.com/59862886/205651758-9152db5f-5f12-4465-8568-86cecfa1781f.png)
(but without a Version Number)

Unfortunately the vuln-scanner doesn't find any vulnerabilities with that Version. 

the vuln-databases are all updated: 

`sqlite3 /var/ossec/queue/vulnerabilities/cve.db "select * from metadata;"

CPEW|Wazuh CPE dictionary|1.0|1.0|2019-10-02T10:56Z||0|0|1655197328
MSU|Microsoft Security Update|1|1|2022-11-30T14:20:40-00|12ce96b3438b3fd0dabb581077b73504943aaa7b5c4fc40505d389276319cd60|107020123|0|1669820077
RHEL5|Red Hat OVAL Patch Definition Merger|3|5.10|2022-12-03T09:49:29||0|0|1670064702
STRETCH|Debian||5.11.2|2022-12-05T07:36:10.188-04:00||0|0|1670234595
BUSTER|Debian||5.11.2|2022-12-05T07:36:45.188-04:00||0|0|1670235139
BULLSEYE|Debian||5.11.2|2022-12-05T07:37:29.188-04:00||0|0|1670235791
FOCAL|Canonical CVE OVAL Generator|1.1|5.11.1|2022-12-05T10:06:08||0|0|1670238205
XENIAL|Canonical CVE OVAL Generator|1.1|5.11.1|2022-12-05T10:06:14||0|0|1670240521
BIONIC|Canonical CVE OVAL Generator|1.1|5.11.1|2022-12-05T10:06:10||0|0|1670240555
TRUSTY|Canonical CVE OVAL Generator|1.1|5.11.1|2022-12-05T10:06:50||0|0|1670241805
RHEL6|Red Hat OVAL Patch Definition Merger|3|5.10|2022-12-05T12:49:43||0|0|1670245131
RHEL7|Red Hat OVAL Patch Definition Merger|3|5.10|2022-12-05T12:50:24||0|0|1670245204
RHEL8|Red Hat OVAL Patch Definition Merger|3|5.10|2022-12-05T12:49:39||0|0|1670245245
JREDHAT|Red Hat Security Data|1.0||2022-12-05 14:01:07||0|0|1670245267`

has anyone an idea what the problem could be?

thanks

**Comment by Dwordcito** [comment]
Hi @martinzeifang 

we do the detection of vulnerable packages through the NVD for Windows. Could you confirm that it is active?

I don't see it on your list.

Do you have a particular CVE for that version of tomcat? I will do local tests.

**Comment by martinzeifang** [comment]
Hi @Dwordcito,

I can confirm that the NVD ist running: 
![image](https://user-images.githubusercontent.com/59862886/205857148-a7ed859a-a2f8-4bf4-959d-6f7201bfc6bb.png)
![image](https://user-images.githubusercontent.com/59862886/205857203-3c50ee87-593c-4153-ab86-2fd69fb54e38.png)

there are many vulnerabilities for this Version: 
https://www.cvedetails.com/vulnerability-list/vendor_id-45/product_id-887/version_id-373488/Apache-Tomcat-4.1.3.html
